source/contracts/architecture.md

```diff
@@ -67,20 +67,22 @@ Before a Safe transaction can be executed, the transaction has to be confirmed b
 
 There are multiple implementations of the Gnosis Safe contract with different methods to check if a transaction has been confirmed by the required owners.
 
-#### GnosisSafePersonalEdition.sol
-This version is targeted at individual users, i.e. only single user would have access to all keys owning a Safe.
-
-Once the required number of confirmations is available `execTransactionAndPaySubmitter` can be called with the sending confirmation signatures. This method will pay the submitter of the transaction for the transaction fees after the Safe transaction has been executed.
+---
+### Gnosis Safe
+#### GnosisSafe.sol
+This contract implements verification of approvals when execution transactions via the contract. 
 
-`execTransactionAndPaySubmitter` expects all confirmations sorted by owner address. This is required to easily validate no confirmation duplicates exist.
+To execute a transaction the method `execTransaction` can be used. To approve a transaction it is necessary to generate and encode the required signatures. 
 
-For more information check out the section about the [Gnosis Safe Personal Edition](./personal_edition.html).
+There are different types of signatures:
 
-#### GnosisSafeTeamEdition.sol
-This version is targeted at teams where each owner is a different user. Each owner has to confirm a transaction by using `approveTransactionWithParameters`. Once the required number of owners has confirmed, the transaction can be executed via `execTransactionIfApproved`. If the sender of `execTransactionIfApproved` is an owner it is not necessary to confirm the transaction before. Furthermore this version doesn't store the nonce in the contract but for each transaction a nonce needs to be specified.
+1. ECDSA signatures generated by Externally Owned Accounts
+1. EIP-1271 based contract signatures
+1. Preferified transaction hash signatures
 
-For more information check out the section about the [Gnosis Safe Team Edition](./team_edition.html).
+For more information check on how the signature types and how to generate and encode the see [Signatures](./signatures.html).
```

Comment:
This sentence sounds weird. Should this be something like `For more information on  signature types including how to generate and encode them, see [Signatures](./signatures.html).`?